compat/visual-composer/siteorigin_widget_vc_template.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$widget_settings = $this->get_widget_settings( $atts );
+$instance = $this->update_widget( $widget_settings['widget_class'], $widget_settings['widget_data'] );
```

Comment:
Yeah, but this isn't necessary. Just something I forgot to remove while developing and testing. :sweat: